src/components/Person/Person.jsx

```diff
@@ -1 +1,41 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const sexPartnerStatus = (sex === 'm') ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+      <p className="Person__age">
+        I am
+        {age}
```

Comment:
There should be a space between the age number and a text
```suggestion
        {`I am ${age}`}
```
<img width="386" alt="image" src="https://user-images.githubusercontent.com/99403198/218438168-9736a95a-f74e-4b98-96a2-ec87a417430f.png">